src/main/java/com/acrolinx/sidebar/utils/XMLLookupUtils.java

```diff
@@ -56,6 +56,11 @@ public static IntRange findOffsetForNodeInXmlStringByXpath(String xmlContent, St
       NodeList nodeList =
           (NodeList) xPath.compile(xpath).evaluate(document, XPathConstants.NODESET);
 
+      if (nodeList.getLength() == 0) { // load document without context awareness
```

Comment:
I think the method name in the next line already conveys this information, so we could omit the comment